db/schema.rb

```diff
@@ -70,6 +70,18 @@
     t.datetime "updated_at", precision: 6, null: false
   end
 
+  create_table "school_device_allocations", force: :cascade do |t|
+    t.bigint "schools_id"
```

Comment:
am wondering should this be `school_id`?

Reply:
ooh, well-spotted, fixed